e2e/tests/server.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
Can't we use a simple node module for that, instead of rewriting the server ourselves?